mypy.ini

```diff
@@ -17,3 +17,9 @@ ignore_missing_imports = True
 
 [mypy-scipy.*]
 ignore_missing_imports = True
+
+[mypy-pandas.*]
+ignore_missing_imports = True
```

Comment:
Do we really need this? We can add the stubs (we did not have pandas typing issues before so I would not remove them)?

Reply:
it might be a versioning thing, but locally it was complaining about the missing stubs. if having the new dependency is fine with you I can add it to the project, and remove this exemption